From 93777093055fe31d4f1ecf28bba79a489d83aac5 Mon Sep 17 00:00:00 2001
From: Andrew Bartlett <abartlet@samba.org>
Date: Sun, 26 May 2013 20:29:19 +1000
Subject: [PATCH] docs: Remove all references to testprns

Based on debian patch documentation2.patch by Christian Perrier <bubulle@debian.org>.

This tool no longer exists in Samba.

Andrew Bartlett
---
 docs-xml/manpages/nmbd.8.xml     |  1 -
 docs-xml/manpages/smb.conf.5.xml |  1 -
 docs-xml/manpages/smbd.8.xml     |  1 -
 docs-xml/using_samba/appd.xml    | 18 ------------------
 docs-xml/using_samba/ch01.xml    |  6 ------
 docs-xml/using_samba/ch07.xml    |  8 --------
 examples/tridge/smb.conf         |  8 --------
 7 files changed, 43 deletions(-)

diff --git a/docs-xml/manpages/nmbd.8.xml b/docs-xml/manpages/nmbd.8.xml
index f666f58..0599ba3 100644
--- a/docs-xml/manpages/nmbd.8.xml
+++ b/docs-xml/manpages/nmbd.8.xml
@@ -266,7 +266,6 @@
 	<manvolnum>8</manvolnum></citerefentry>, <citerefentry><refentrytitle>smb.conf</refentrytitle>
 	<manvolnum>5</manvolnum></citerefentry>, <citerefentry><refentrytitle>smbclient</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>testparm</refentrytitle>
-	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>testprns</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>, and the Internet 
 	RFC's <filename>rfc1001.txt</filename>, <filename>rfc1002.txt</filename>. 
 	In addition the CIFS (formerly SMB) specification is available 
diff --git a/docs-xml/manpages/smb.conf.5.xml b/docs-xml/manpages/smb.conf.5.xml
index 44411b0..dd4f858 100644
--- a/docs-xml/manpages/smb.conf.5.xml
+++ b/docs-xml/manpages/smb.conf.5.xml
@@ -809,7 +809,6 @@ chmod 1770 /usr/local/samba/lib/usershares
 	<manvolnum>8</manvolnum></citerefentry>, <citerefentry><refentrytitle>smbclient</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>nmblookup</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>testparm</refentrytitle>
-	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>testprns</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>.</para>
 </refsect1>
 
diff --git a/docs-xml/manpages/smbd.8.xml b/docs-xml/manpages/smbd.8.xml
index 98e76fb..0d246cd 100644
--- a/docs-xml/manpages/smbd.8.xml
+++ b/docs-xml/manpages/smbd.8.xml
@@ -421,7 +421,6 @@
 	<manvolnum>8</manvolnum></citerefentry>, <citerefentry><refentrytitle>smb.conf</refentrytitle>
 	<manvolnum>5</manvolnum></citerefentry>, <citerefentry><refentrytitle>smbclient</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>testparm</refentrytitle>
-	<manvolnum>1</manvolnum></citerefentry>, <citerefentry><refentrytitle>testprns</refentrytitle>
 	<manvolnum>1</manvolnum></citerefentry>, and the 
 	Internet RFC's	<filename>rfc1001.txt</filename>, <filename>rfc1002.txt</filename>. 
 	In addition the CIFS (formerly SMB) specification is available 
diff --git a/docs-xml/using_samba/appd.xml b/docs-xml/using_samba/appd.xml
index a3a23f8..018e590 100644
--- a/docs-xml/using_samba/appd.xml
+++ b/docs-xml/using_samba/appd.xml
@@ -1315,24 +1315,6 @@ received 6 names
 
 
 
-<sect2 role="" label="D.1.11" id="appd-SECT-1.11">
-<title>testprns</title>
-
-
-<para>The<indexterm id="appd-idx-993761-0"><primary>testprns program</primary></indexterm>
-<indexterm id="appd-idx-993761-1"><primary>printers</primary><secondary>names</secondary><tertiary>checking</tertiary></indexterm> <emphasis>testprns</emphasis> program checks a specified printer name against the system printer capabilities (<filename>printcap</filename>) file. Its command line is:</para>
-
-
-<programlisting>testprns <replaceable>printername</replaceable> [<replaceable>printcapname</replaceable>]</programlisting>
-
-
-<para>If the <literal>printcapname</literal> isn't specified, Samba attempts to use one located in the <filename>smb.conf</filename> file. If one isn't specified there, Samba will try <filename>/etc/printcap</filename>. If that fails, the program will generate an error.</para>
-</sect2>
-
-
-
-
-
 <sect2 role="" label="D.1.12" id="appd-SECT-1.12">
 <title>rpcclient</title>
 
diff --git a/docs-xml/using_samba/ch01.xml b/docs-xml/using_samba/ch01.xml
index ca8bc13..01d7791 100644
--- a/docs-xml/using_samba/ch01.xml
+++ b/docs-xml/using_samba/ch01.xml
@@ -1375,12 +1375,6 @@ SIMPLE           &lt;1E&gt;  GROUP       Registered
 </varlistentry>
 
 
-<varlistentry><term>testprns</term>
-<listitem><para>A program that tests whether various printers are recognized by the <filename>smbd</filename> daemon</para></listitem>
-</varlistentry>
-</variablelist>
-
-
 <para>Each significant release of Samba goes through a significant exposure test before it's announced. In addition, it is quickly updated afterward if problems or unwanted side-effects are found. The latest stable distribution as of this writing is Samba 2.0.5, the long-awaited production version of Samba 2.0. This book focuses on the functionality supported in Samba 2.0, as opposed to the older 1.9.<emphasis>x</emphasis> versions of Samba, which are now obsolete.</para>
 </sect1>
 
diff --git a/docs-xml/using_samba/ch07.xml b/docs-xml/using_samba/ch07.xml
index 307cab7..988aab8 100644
--- a/docs-xml/using_samba/ch07.xml
+++ b/docs-xml/using_samba/ch07.xml
@@ -306,14 +306,6 @@ lppause command:
 	public: true</programlisting>
 
 
-<para>Second, try the command <literal>testprns</literal> <replaceable>printername</replaceable>. This is a simple program that verifies that the specified printer is available in your <emphasis>printcap</emphasis> file. If your <emphasis>printcap</emphasis> file is not in the usual place, you can specify its full pathname as the second argument to the <emphasis>testprns</emphasis> command:</para>
-
-
-<programlisting># testprns lp /etc/printcap
-Looking for printer lp in printcap file /etc/printcap
-Printer name lp is valid.</programlisting>
-
-
 <para>Next, log on as the guest user, go to the spooling directory, and ensure that you can print using the same command that <emphasis>testparm</emphasis> says Samba will use. As mentioned before, this will tell you if you need to change the guest account, as the default account may not be allowed to print.</para>
 
 
diff --git a/examples/tridge/smb.conf b/examples/tridge/smb.conf
index a2f269f..4aa40d8 100644
--- a/examples/tridge/smb.conf
+++ b/examples/tridge/smb.conf
@@ -31,14 +31,6 @@
    print ok = yes
    print command = xmenu -heading "%s" OK&
 
-[testprn]
-   comment = Test printer
-   path = /tmp
-   user = susan
-   print ok = yes
-   print command = cp %s /tmp/smb.%U.prn
-   lpq command = cat /tmp/xxyz
-
 [amd]
    comment = amd area
    path = /mount
-- 
1.7.11.7

